### PR TITLE
Fixes #3937: When executed from Special Functions, Flight Reset cause…

### LINKED ISSUE #3937: SF "reset flight" warning display issues
Horus, git checkout

When I use the SF "reset flight" my radio locks up.
The power LED turns red and the radio will not be freed until I press a button at the thumbwheel.

### PR DESCRIPTION
**Pull request by projectkk2glider**
…d a GUI lockup

Closes #3937

### CODE REVIEW

File: radio/src/opentx.h [hunk 1]
```diff
@@ -1347,8 +1347,13 @@ void clearMFP();
 extern uint8_t requiredSpeakerVolume;
 #endif
 
-#if defined(PCBTARANIS)
-extern uint8_t requestScreenshot;
+#if defined(CPUARM)
+enum MainRequest {
+  RequestScreenshot,
```
Comment (bsongis) [inline radio/src/opentx.h]: Rather name them like constants: ENUM_NAME.

Comment (projectkk2glider) [inline radio/src/opentx.h]: You mean REQUEST_SCREENSHOT?

Comment (bsongis) [inline radio/src/opentx.h]: Yes, I just updated the coding style wiki page, it was not written

**Comment by kilrah** [comment]
All good for me, works as expected. Mixer still runs and updates servo positions while warnings are displayed, but that's probably the best way to go about it. Thanks!

**Pull request merged**